benches/benches/transaction_throughput.rs

```diff
@@ -90,6 +93,9 @@ where
             test_builder.utxo_validation = true;
             test_builder.gas_limit = Some(10_000_000_000);
             test_builder.block_size_limit = Some(1_000_000_000_000);
+            test_builder.database_type = DbType::RocksDb;
+            test_builder.max_database_cache_size = Some(16 * 1024 * 1024 * 1024);
```

Comment:
Maybe we can just set database config here instead of passing each field one by one?

Reply:
Done